### PR TITLE
Task runner

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -8,7 +8,6 @@ parameterized = "==0.7.0"
 black = "==19.10b0"
 prospector = "==1.1.6.2"
 bandit = "*"
-coverage = ">=4.5.4"
```
Comment (charlesluokb) [inline Pipfile]: why `coverage` is removed?

File: docker-compose.yml [hunk 2]
```diff
@@ -29,3 +29,40 @@ services:
     expose:
       - 8000
     command: pipenv run ./manage.py runserver 0.0.0.0:8000
+
+  mail:
+    build: ./mailserver
```
Comment (charlesluokb) [inline docker-compose.yml]: wondering if we should check in the folder ./mailserver for sack of building a full local env?

Comment (cstobart) [reply docker-compose.yml]: I was thinking of maybe using it as a submodule

File: Pipfile (hunk 1 above)
Comment (cstobart) [reply Pipfile]: mistakingly

File: docker-compose.yml (hunk 2 above)
Comment (charlesluokb) [reply docker-compose.yml]: I have added the folder `/mailserver`, and updated docker compose file accordingly. If you follow README, then you will have four containers running locally
- lite-hmrc-intg
- lite-hmrc-postgres
- lite-hmrc-ditmail
- lite-hmrc-spiremail

I have also updated .dockerignore to exclude /database and /mailserver when building image, hence we do not need to make /mailserver as a submodule

**Pull request merged**